src/workerd/api/container.c++

```diff
@@ -52,6 +52,12 @@ void Container::start(jsg::Lock& js, jsg::Optional<StartupOptions> maybeOptions)
   running = true;
 }
 
+jsg::Promise<void> Container::setInactivityTimeout(jsg::Lock& js, kj::Date scheduledTime) {
```

Comment:
I am confused by the "scheduled time" idea.  I thought that `setInactivityTimeout` was going to take a duration rather than a timestamp.

Reply:
changed to duration